tests/templates/kuttl/logging/40-create-configmap-with-prepared-logs.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: >
+      kubectl create configmap prepared-logs
+      --from-file=prepared-logs.py.json
+      --namespace=$NAMESPACE
```

Comment:
Any reason to not just put the logs straight into the YAML instead?

Reply:
I wanted to ask the same question for the Spark PR ... but then I saw the XML there :)

Reply:
Yes, most of the log files are XML files. The line endings (`\n` and `\r\n`) are important. And in one case, there are ANSI escape sequences.

It is easy to just capture the content of the deployed ConfigMap and put it into a YAML file. But it is then very hard to read or change this file.